docs/io/visualization/index.rst

```diff
@@ -27,4 +27,5 @@ Jupyter Notebooks, making data exploration much easier.
     :maxdepth: 2
 
     using_widgets
-    Generating Widgets in a Notebook <generating_widgets>
\ No newline at end of file
+    Generating Widgets in a Notebook <generating_widgets>
+    abundance_widget
```

Comment:
```suggestion
    Generating Custom Abundance Widget <abundance_widget>
    Generating Data Exploration Widgets <generating_widgets>
```
For now let's keep it this way.

Reply:
Also can you edit the title of generating_widgets.ipynb to same?

Reply:
I edit the title of both `generating_widgets.ipynb` and `abundance_widget.ipynb`. I also clear the output and commented the cell that generates `shell_info_widget` from HDF file. Please tell me if you don't want to do that.

Reply:
Since you edited the title there was no need of mentioning explicit title in TOC but NVM